packages/react-dom-bindings/src/server/ReactFizzConfigDOMLegacy.js

```diff
@@ -92,8 +92,6 @@ export function createRenderState(
     undefined,
     undefined,
     undefined,
-    undefined,
-    undefined,
```

Comment:
Same here. Were we sending one less already?

Reply:
Yeah I checked on those, I believe we were sending too many undefineds before. I confirmed the signature has six args and there are six inputs when 2 are removed